include/aspect/stokes_matrix_free.h

```diff
@@ -305,7 +305,10 @@ namespace aspect
       virtual void evaluate_material_model()=0;
 
       /**
-       * Add correction to system RHS for non-zero boundary condition.
+       * Add correction to system RHS for non-zero boundary condition. For more information
+       * on exactly what this correction is and why it is computed, see the deal.II tutorial
+       * step 50 section "LaplaceProblem::assemble_rhs()":
+       * https://www.dealii.org/developer/doxygen/deal.II/step_50.html#LaplaceProblemassemble_rhs
        */
```

Comment:
This may be too lazy.. Should I add in a short description about the residual computation (like in step-50 documentation)?

Reply:
I think this is good as is.